### ISSUE TITLE
stream via Bluetooth fearture

### ISSUE THREAD

**Issue by syrakozz** [tumtumtum/StreamingKit#181]
can you add a Bluetooth feature so we can stream via Bluetooth car

**Comment by syrakozz** [comment]
@tumtumtum ,
any idea about this

**Comment by hutchy** [comment]
Is there a reason you don't simply just want to use AirPlay?

https://developer.apple.com/airplay/

**Comment by syrakozz** [comment]
@hutchy , becasue i need to put airplay device in the car to connect the iphone to it, right ?
or there is another way, without device, just iphone and car ?

**Comment by hutchy** [comment]
I don't think this is an **issue** related to this repo, and more a question about the iPhone technically. In my experience, in the UK, cars that have Bluetooth technology do not support audio - simply phone calls etc.

**Comment by Adirael** [comment]
Nowadays most cars support both phone calls an audio streams, but I'm not sure if it's related to this project or not, I don't think it does. 

My car's audio system is detected by iOS as an external speaker and I can just select it.
